templates/website/package.json

```diff
@@ -11,7 +11,7 @@
     "dev:prod": "cross-env NODE_OPTIONS=--no-deprecation rm -rf .next && pnpm build && pnpm start",
     "generate:importmap": "cross-env NODE_OPTIONS=--no-deprecation payload generate:importmap",
     "generate:types": "cross-env NODE_OPTIONS=--no-deprecation payload generate:types",
-    "ii": "cross-env NODE_OPTIONS=--no-deprecation pnpm --ignore-workspace install",
```

Comment:
Why is this command being removed?

Reply:
because now you have it in the root package.json (as part of `template:blank` and `template:website`).
I didn't think it felt right to add scripts to the users in the template that are intended to be used by us in the monorepo if we can run them from the outside. This way I think it is more convenient for them as well as for us.

Reply:
Okay, that part makes sense. 

Regarding the new `preinstall` script, though. I'm a bit concerned this has some ramifications. Feels like we should avoid hijacking that script.

- What if the dev wants to clear their node_modules and re-run install? This would overwrite their now possibly-modified .env file.
- What if the dev builds their application in the same environment they run it in? This would overwrite the file in a production environment.

Reply:
The first will not happen because the second time you run the script there is no .env.example and therefore the script does nothing.
As for your second point, it won't happen either unless someone makes the first commit before installing. And even if that happens, I'm not sure it would be problematic in a build environment.

Still, I think it would actually be best to have CPA create the .env, remove the .env.example, the preinstall script, and [the third method of “cloning”](https://github.com/payloadcms/payload/blob/main/templates/website/README.md?plain=1#L43-49) from the readme.  So that the 2 ways to initialize the template are CPA and Payload Cloud. Either way, I don't think anyone is using or would want to use the very long git command. 

I'm going to see this weekend if it's easy to make. Although if it takes a long time, I think this PR is already an improvement (after fixing the Windows problem of course).